packages/utils/src/objects.ts

```diff
@@ -30,7 +30,7 @@ export function toExpectedCase(
 }
 
 function isObjectObject(val: unknown): boolean {
-  return val != null && typeof val === "object" && Array.isArray(val) === false;
+  return val !== null && typeof val === "object" && Array.isArray(val) === false;
```

Comment:
diff is safe. `isObjectObject` will continue to return false if `val` is `null` or `undefined`